### ISSUE TITLE
Error while compiling main.tex

### ISSUE THREAD

**Issue by Suzukazole** [wmboyles/Math-Summaries#30]
Hey, I had pulled your recent changes and tried compiling it, but there were lots of errors. (on Diffeq and on Multicalc)
This one seems to be the root cause.

``
[91])) (./appendix/appendixA.tex [92]
Appendix A.
(./appendix/laplaceTable/laplaceTable.tex
Use of \@xnext doesn't match its definition.
\@freelist ->,
              \bx@A ,,\bx@B ,,\bx@C ,,\bx@D ,,\bx@E ,,\bx@F ,,\bx@G ,,\bx@H ...
l.5 \begin{table}[h]
``

**Comment by wmboyles** [comment]
I was unable to replicate the problem. I went ahead and changed `\begin{table}[h]` to `\begin{table}[H]` in `appendixA.tex` because I remember this causing a problem in the past. Lt me know if this fixes anything. I don't think this change should explain your issues with multicalc because the appendix is only in diffeq.

I went ahead and recompiled all of the PDFs to be sure, but I wasn't able to replicate the problem.
